config/default/kustomization.yaml

```diff
@@ -12,18 +12,18 @@ namePrefix: kueue-
 #commonLabels:
 #  someName: someValue
 
-bases:
-- ../crd
-- ../rbac
-- ../manager
+resources:
+- ../components/crd
+- ../components/rbac
+- ../components/manager
+- ../components/internalcert
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-- ../webhook
+# ../components/crd/kustomization.yaml
+- ../components/webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-# - ../certmanager
-- ../internalcert
+# - ../components/certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+#- ../components/prometheus
```

Comment:
also, do we have docs explaining how to set metrics up (at least point to kubebuilder docs if it is exactly the same)?

Reply:
Yes, this creates a CR for a CRD defined by prometheus-operator. So the installation would fail if prometheus-operator was not installed first.

I'm working on the documentation as my next PR.